build.rs

```diff
@@ -0,0 +1,14 @@
+// inspired by https://github.com/p2panda/meli/pull/21/files
+// meli is licensed under GNU AFFERO GENERAL PUBLIC LICENSE
```

Comment:
Which means we're not allowed to copy/paste the code in an MIT project :(

Reply:
it's an adapted version of their fix to adapt to our use case (i.e. default path `$HOME/Android/Sdk` instead of adaptable path)